CHANGELOG.md

```diff
@@ -28,8 +28,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
     connections to the database, and is intended to be a temporary solution
     until Cumulus has been updated to import the RDS rds-ca-rsa2048-g1 CA bundles in Lambda environments.
     See [CUMULUS-3724](https://bugs.earthdata.nasa.gov/browse/CUMULUS-3724).
+- **CUMULUS-3906**
+  - Bumps example ORCA deployment to version v10.0.0. This version is necessary to support
+    Aurora Serverless V2.
```

Comment:
Is Aurora Serverless V2 necessary to support orca v10.0.0 or cumulus with serverless v2 has to use orca v10.0.0?

Reply:
@jennyhliu Pushed a small edit and removed the required language.